sample_apps/run_example.sh

```diff
@@ -135,6 +135,7 @@ SampleApps=( "simple_app"
              "simple_app_under_app_service"
              "simple_app_under_keystone"
              "simple_app_under_islet"
+             "simple_app_python"
```

Comment:
Most of the changes in this script is to inject this new app in existing switch / case code-flow.